art-bot.py

```diff
@@ -379,5 +379,19 @@ def run(debug):
     handler.start()
 
 
+def signal_handler(_signo, _stack_frame):
+    """
+    Function to handle SIGTERM signal sent during pod termination. Check if check nightly color command is still running.
+    If it is, report to the user saying that they have re-run the command.
+    """
+    slack_objects = variables.active_slack_objects
+    if slack_objects:
+        for slack_object in slack_objects:
+            plain_text = extract_plain_text({"data": slack_object.event})
+            slack_object.say(f"Uh oh... the pod was restarted. Please run *'{plain_text}'* again.")
```

Comment:
"the pod" seems like a non sequitur here if you know nothing about art-bot infra - how about
```suggestion
            slack_object.say(f"Uh oh... art-bot is restarting. Please try *'{plain_text}'* again.")
```

Reply:
Fixed in https://github.com/openshift-eng/art-bot/pull/47/commits/d6684750d95a85bee51ee9ec564d28f9e58ca02a